### PR TITLE
Fix duplicates in community context builder

### PR DESCRIPTION
**Pull request by AlonsoGuevara**
## Description

kudos to @jarlor for #1088

## Related Issues

[Reference any related issues or tasks that this pull request addresses.]

## Proposed Changes

[List the specific changes made in this pull request.]

## Checklist

- [ ] I have tested these changes locally.
- [ ] I have reviewed the code changes.
- [ ] I have updated the documentation (if necessary).
- [ ] I have added appropriate unit tests (if applicable).

## Additional Notes

[Add any additional notes or context that may be helpful for the reviewer(s).]

### CODE REVIEW

File: graphrag/query/context_builder/community_context.py [hunk 1]
```diff
@@ -140,7 +140,12 @@ def _cut_batch() -> None:
         if len(record_df) == 0:
             return
         current_context_text = record_df.to_csv(index=False, sep=column_delimiter)
-        all_context_text.append(current_context_text)
+        if (not all_context_text) and single_batch:
```
Comment (andresmor-ms) [inline graphrag/query/context_builder/community_context.py:143]: are these parentheses required?

File: graphrag/query/context_builder/community_context.py [hunk 2]
```diff
@@ -140,7 +140,12 @@ def _cut_batch() -> None:
...
+            all_context_text.append(
+                f"-----{context_name}-----" + "\n" + current_context_text
```
Comment (andresmor-ms) [inline graphrag/query/context_builder/community_context.py:145]: if using f-string you could do `f"-----{context_name}-----\n{current_context_text}"`

File: graphrag/query/context_builder/community_context.py (hunk 1 above)
Comment (AlonsoGuevara) [reply graphrag/query/context_builder/community_context.py:143]: Nope.
Pretty sure I had those removed, let me double check

Comment (AlonsoGuevara) [reply graphrag/query/context_builder/community_context.py]: Updated

File: graphrag/query/context_builder/community_context.py (hunk 2 above)
Comment (AlonsoGuevara) [reply graphrag/query/context_builder/community_context.py]: Updated

**Review by andresmor-ms — APPROVED** [review]

**Pull request merged**